src/main/java/net/apnic/rdap/stats/scraper/DelegatedStatsScraper.java

```diff
@@ -192,35 +120,17 @@ private void handleIPRecord(IPRecord record, ResourceStore store,
      *
      * @return Future containing a stream of the data recieved from the server
```

Comment:
May want to update the javadocs to reflect the signature change.

Reply:
Updated.